data/scripts/cloud-remediation.sh

```diff
@@ -0,0 +1,3 @@
+# Run the image hardening script and then remove the package after completion
+/opt/pct/cloud-remediation.sh
```

Comment:
This makes the assumption that the script is set to executable, which is fair to a certain extend. That said it might be safer to specify the interpreter, `/usr/bin/bash` if this is a bash script, or `/usr/bin/sh` or whatever.

The other question we need to answer if we want the image build to fail if application of remediation fails. Then we'd have to check the exit condition and exit.

Reply:
I can add the interpreter path to the call.  Do you want to make the call to the image failing if remediation fails or do we discuss that in a larger group setting?

Reply:
I think it should fail, we are trying to produce a hardened image after all. @jgleissner thoughts?

Reply:
see if this is works for what you are expecting